limits.c

```diff
@@ -409,6 +405,6 @@ void limits_force_servo(){
   
   plan_sync_position(); // Sync planner position to current machine position for pull-off move.
 
-  limits.mag_gap_check = MAG_GAP_CHECK_ENABLE; // Start checking magazine gaps on carousel again
+  limits.mag_gap_check = settings.mag_gap_enabled; // Start checking magazine gaps on carousel again
```

Comment:
limits.mag_gap_check seems to be just an alias no?  Do we need this intermediary or could we just check settings.mag_gap_enabled?